android/anc/src/main/java/org/smartregister/fhircore/anc/ui/details/bmicompute/BmiQuestionnaireActivity.kt

```diff
@@ -124,9 +143,10 @@ class BmiQuestionnaireActivity : QuestionnaireActivity() {
           questionnaireResponse,
           patientId,
           encounterID,
-          height,
           weight,
-          computedBMI
+          height,
+          computedBMI,
+          isUnitModeMetric
```

Comment:
Add name to function parameters to improve readability

Reply:
done